Automattic/wp-calypso
# Notifications: Add translations to the app

Fixes https://github.com/Automattic/dotcom-forge/issues/8684

## Proposed Changes
Move translation files to the app itself.

## Why are these changes being made?
Language files should be inside the app.

## Testing Instructions

- Go to '/apps/notifications'
- Run `yarn build --sync`
- Sandbox `widgets.wp.com`
- Click on the bell icon.
- Check the Network tab in the Developers console
- The notifications language should load from `widgets.wp.com/notifications/languages/$LANG`
- Check for regressions
